src/test/java/net/mcreator/integration/generator/GTJSONTriggersBlocks.java

```diff
@@ -40,6 +40,7 @@
 
 public class GTJSONTriggersBlocks {
 
+	@SuppressWarnings("DuplicateBranchesInSwitch")

```

Comment:
I had a brainfart here, you can indeed merge those since the block you put the ouput block in is the same